hudi-support/extensions/src/test/java/io/onetable/hudi/sync/TestOneTableSyncTool.java

```diff
@@ -77,8 +77,6 @@ public static void initSpark() {
     spark = SparkSession.builder().config(sparkConf).getOrCreate();
   }
 
-  @ParameterizedTest
```

Comment:
TODO: Need help in fixing this test. This test is currently failing with an error related to serialization.
`Caused by: java.lang.AbstractMethodError: Receiver class org.apache.spark.sql.adapter.Spark3_4Adapter does not define or inherit an implementation of the resolved method 'abstract org.apache.spark.sql.avro.HoodieAvroSerializer createAvroSerializer(org.apache.spark.sql.types.DataType, org.apache.avro.Schema, boolean)' of interface org.apache.spark.sql.hudi.SparkAdapter.
	at org.apache.hudi.AvroConversionUtils$.createInternalRowToAvroConverter(AvroConversionUtils.scala:59)`

Reply:
I'll take a look at this one but there are more tests failing.